### DISCUSSION TITLE
How get mean.bin (from captcha recognition)?

### DISCUSSION THREAD

**Discussion by jasperDD** [apache/incubator-mxnet#14074] [Q&A]
I have question about this great post
https://github.com/apache/incubator-mxnet/blob/master/example/captcha/mxnet_captcha.R
i downloaded data, but there is no in archive mean.bin
So i just want read rec file, lile show here 
https://github.com/apache/incubator-mxnet/blob/master/R-package/tests/testthat/test_io.R
Where can i find mean.bin

**Comment by andrewfayres** [comment]
@mxnet-label-bot add [Question, R]

@jasperDD If you haven't already, you should try asking this question at discuss.mxnet.io. Your question will get more visibility in the community.

**Comment by Ishitori** [comment]
It seems the file is lost, but you can recreate it. Basically, mean.bin is a mean image: every pixel of this image is a mean value of pixels on that position of all training images. That mean image is used to normalize every image during training and prediction.

You can recreate this file by:
- Take all training images tensors and sum them up
- Divide resulting tensor to number of training images 
- Resulting tensor is a mean image - save it to file with mean.bin file name.

Also, if you do all of this and it will work, please contribute back to S3!

**Comment by piyushghai** [comment]
@jasperDD Were you able to re-generate the mean.bin files based on @Ishitori 's suggestions ?
